### PR TITLE
Use asahi-alarm master key

### PR DESCRIPTION
**Pull request by mkurz**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,8 +28,10 @@ You can delete `/etc/pacman.d/mirrorlist.asahi` now.
 
 Remove the old keyring package:
 
+Install the new keyring package (will replace the legacy `asahilinux-keyring`):
+
 ```
-sudo pacman -Rdd asahilinux-keyring
+sudo pacman -S asahi-alarm-keyring
```
Comment (joske) [inline README.md:34]: is this right? The PKGBUILD does not specify conflicts (yet - we should probably do this though)

File: README.md [hunk 2]
```diff
@@ -7,8 +7,8 @@ These packages are for the Arch linux ARM variant of Asahi Linux.
 First import the signing key
 
 ```bash
-sudo pacman-key --recv-keys 1F35A37F9D85CBC2B74F595E37CE6162C969C50E
-sudo pacman-key --lsign-key 1F35A37F9D85CBC2B74F595E37CE6162C969C50E
+sudo pacman-key --recv-keys 12CE6799A94A3F1B5DDFFE88F576553597FB8FEB
```
Comment (joske) [inline README.md:10]: how will this work? Will we both have the private key for this?

Comment (joske) [reply README.md:10]: oh I just saw the secrets repo 👍

Comment (joske) [reply README.md:10]: so will need to make a new release signed with that key. Also we'll have to update the `asahi-alarm-keyring` repo

File: README.md (hunk 1 above)
Comment (joske) [reply README.md:34]: I see you already did 👍

File: README.md (hunk 2 above)
Comment (mkurz) [reply README.md:10]: @joske 
- https://github.com/asahi-alarm/asahi-alarm-keyring/pull/1

**Pull request merged**